### PR TITLE
update installation instructions

### PR DESCRIPTION
**Pull request by nikolas-burkoff**
Part of https://github.com/insightsengineering/teal/issues/668

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,12 +15,14 @@ The package provides:
 
 ## Installation
 
-It is recommended that you [create and use a Github PAT](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token) to install the latest version of this package. Once you have the PAT, run the following:
+For releases from August 2022 it is recommended that you [create and use a Github PAT](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token) to install the latest version of this package. Once you have the PAT, run the following:
```
Comment (dinakar29) [inline README.md:18]: This is apt.

**Review by dinakar29 — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -15,12 +15,14 @@ The package provides:
...
-if (!require("devtools")) install.packages("devtools")
-devtools::install_github("insightsengineering/osprey@*release")
+if (!require("remotes")) install.packages("remotes")
+remotes::install_github("insightsengineering/osprey@*release")
 ```
 
-In order to run many of the examples you will also need to install the [`scda`](https://insightsengineering.github.io/scda) package.
+A stable release of all `NEST` packages from June 2022 is also available [here](https://github.com/insightsengineering/depository#readme).
+
+In order to run many of the examples you will also need to install the [`scda`](https://github.com/insightsengineering/scda) package.
```
Comment (nikolas-burkoff) [inline README.md:28]:
```suggestion
In order to run many of the examples you will need to install the [`scda`](https://github.com/insightsengineering/scda) package.
```

File: README.md [hunk 3]
```diff
@@ -15,12 +15,14 @@ The package provides:
...
-if (!require("devtools")) install.packages("devtools")
-devtools::install_github("insightsengineering/osprey@*release")
+if (!require("remotes")) install.packages("remotes")
+remotes::install_github("insightsengineering/osprey@*release")
 ```
 
-In order to run many of the examples you will also need to install the [`scda`](https://insightsengineering.github.io/scda) package.
+A stable release of all `NEST` packages from June 2022 is also available [here](https://github.com/insightsengineering/depository#readme).
+
+In order to run many of the examples you will need to install the [`scda`](https://github.com/insightsengineering/scda) package.
```
Comment (nikolas-burkoff) [inline README.md:28]:
```suggestion
In order to run many of the examples you will also need to install the [`scda`](https://github.com/insightsengineering/scda) package.
```

**Pull request merged**